script.js

```diff
@@ -1,4 +1,66 @@
+var Scissors = "scissors";
+var Paper = "paper";
+var Stone = "stone";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var opponentNumber = randomNumber();
```

Comment:
To make your code eve cleaner: Use verbs (e.g. get, create, etc.) to name your helper functions. That way, this line will be even easier to read. I.e. `var opponentNumber= getOpponentNumber()` --> Notice how, semantically, the variable name for the helper function makes sense with respect to the name of the variable that you are assigning the result to? 

This also benefits you cos you don't have to think of so many different variable names.